libs/portal/shared/features/form/ui/month-year-field.tsx

```
@@ -157,7 +157,13 @@
                                 setMonthOpen(false)
                               }}
                             >
-                              {option.label}
+                              <button
+                                type="button"
+                                role="menu"
+                                aria-label={option.label}
+                              >
```

Biome (1.9.4)

[error] Interactive elements should not be assigned non-interactive roles.

WAI-ARIA roles should not be used to convert an interactive element to a non-interactive element.
Wrap your interactive element in a <div> with the desired role or put the content inside your interactive element.
Unsafe fix: Remove the role attribute.


(lint/a11y/noInteractiveElementToNoninteractiveRole)